libmattermost.c

```diff
@@ -3055,7 +3056,12 @@ mm_got_users_of_room(MattermostAccount *ma, JsonNode *node, gpointer user_data)
 	PurpleChatConversation *chatconv = purple_conversations_find_chat(ma->pc, g_str_hash(channel_id));
 	JsonObject *obj = json_node_get_object(node);
 	PurpleGroup *default_group = mm_get_or_create_default_group();
-	
+	gboolean found_myself = FALSE;
+
+	if (chatconv != NULL) {
+		found_myself = purple_chat_conversation_has_user(chatconv, ma->self_username);

```

Comment:
We can be listed in the room twice: 1 - from user_added websocket callback, 2 - from code below....lets avoid that